### DISCUSSION TITLE
VPN ile sınırsız referans kasma

### DISCUSSION THREAD

**Discussion by Omertastan2036** [IBAX-io/go-ibax#918] [Soru Cevap - Türkçe]
Ya millet VPN ile fake hesaplar açıp referans kasıyor bunlar tespit edilebiliyor mu. Yoksa bu proje o kadar da güven verici değil mk

**Comment by Yusufknk** [comment]
Yinelenen ve bot işlemler diskalifiye edilecektir merak etmeyin.

**Reply by Omertastan2036** [reply]
VPN kullanıyor bot olup olmadigini nereden bilsin internet id si farklı

**Comment by zukenberger** [comment]
Kanka ileride kyc isterler. O zaman botlar temizlenir.

**Reply by Omertastan2036** [reply]
> Kanka ileride kyc isterler. O zaman botlar temizlenir.

Şimdiden istesin knk daha mantıklı değil mi?
